src/Microsoft.Azure.ServiceBus.KeyVault/Constants.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.ServiceBus.KeyVault
+{
+    internal static class Constants
```

Comment:
`Constants` or `Headers`? These are used for headers looks like.
(Yes, they are implemented as constants, but that's just a technical detail).